core/src/main/java/io/seata/core/store/db/sql/log/AbstractLogStoreSqls.java

```diff
@@ -0,0 +1,208 @@
+
+    /**
+     * The constant QUERY_GLOBAL_TRANSACTION.
+     */
+    public static final String QUERY_GLOBAL_TRANSACTION = "select " + ALL_GLOBAL_COLUMNS + " from "
+            + GLOBAL_TABLE_PLACEHOLD + " where " + ServerTableColumnsName.GLOBAL_TABLE_XID + " = ?";
+
+    /**
+     * The constant QUERY_GLOBAL_TRANSACTION_ID.
+     */
+    public static final String QUERY_GLOBAL_TRANSACTION_BY_ID = "select " + ALL_GLOBAL_COLUMNS + " from "
+            + GLOBAL_TABLE_PLACEHOLD + " where " + ServerTableColumnsName.GLOBAL_TABLE_TRANSACTION_ID + " = ?";
+
+    /**
+     * The constant DELETE_BRANCH_TRANSACTION_BY_BRANCH_ID.
+     */
+    public static final String DELETE_BRANCH_TRANSACTION_BY_BRANCH_ID = "delete from " + BRANCH_TABLE_PLACEHOLD
+            + " where " + ServerTableColumnsName.BRANCH_TABLE_XID + " = ? and " + ServerTableColumnsName.BRANCH_TABLE_BRANCH_ID
+            + " = ?";
+
+    /**
+     * The constant DELETE_BRANCH_TRANSACTION_BY_XID.
+     */
+    public static final String DELETE_BRANCH_TRANSACTION_BY_XID = "delete from " + BRANCH_TABLE_PLACEHOLD
+            + " where " + ServerTableColumnsName.BRANCH_TABLE_XID + " = ?";
+
+
+    /**
+     * The constant QUERY_BRANCH_TRANSACTION.
+     */
+    public static final String QUERY_BRANCH_TRANSACTION = "select " + ALL_BRANCH_COLUMNS + " from "
+            + BRANCH_TABLE_PLACEHOLD + " where " + ServerTableColumnsName.BRANCH_TABLE_XID + " = ? order by "
+            + ServerTableColumnsName.BRANCH_TABLE_GMT_CREATE + " asc";
+
+    /**
+     * The constant QUERY_BRANCH_TRANSACTION_XIDS.
+     */
+    public static final String QUERY_BRANCH_TRANSACTION_XIDS = "select " + ALL_BRANCH_COLUMNS + " from "
+            + BRANCH_TABLE_PLACEHOLD + " where " + ServerTableColumnsName.BRANCH_TABLE_XID + " in (" + PRAMETER_PLACEHOLD + ") order by "
+            + ServerTableColumnsName.BRANCH_TABLE_GMT_CREATE + " asc";
+
+    /**
+     * The constant CHECK_MAX_TRANS_ID.
+     */
+    public static final String QUERY_MAX_TRANS_ID = "select max(" + ServerTableColumnsName.GLOBAL_TABLE_TRANSACTION_ID
+            + ") from " + GLOBAL_TABLE_PLACEHOLD + " where " + ServerTableColumnsName.GLOBAL_TABLE_TRANSACTION_ID
+            + " < ? and " + ServerTableColumnsName.GLOBAL_TABLE_TRANSACTION_ID + " > ?";
+
+    /**
+     * The constant CHECK_MAX_BTANCH_ID.
+     */
+    public static final String QUERY_MAX_BTANCH_ID = "select max(" + ServerTableColumnsName.BRANCH_TABLE_BRANCH_ID
+            + ") from " + BRANCH_TABLE_PLACEHOLD + " where " + ServerTableColumnsName.BRANCH_TABLE_BRANCH_ID + " < ? and "
+            + ServerTableColumnsName.BRANCH_TABLE_BRANCH_ID + " > ?";
+
+    @Override
+    public String getInsertGlobalTransactionSQL(String globalTable) {
+        throw new NotSupportYetException("unknown dbType:" + CONFIG.getConfig(ConfigurationKeys.STORE_DB_TYPE));
```

Comment:
Suggest:
return getInsertGlobalTransactionSQL().replace(GLOBAL_TABLE_PLACEHOLD, globalTable);
String getInsertGlobalTransactionSQL()();
Same for other sql

Reply:
OK

Reply:
fixed